source/types/deploys.json

```diff
@@ -1,4 +1,6 @@
 {
-  "1": "0xa42d613D8B8B6cB9507F46A5dd3FD98276570FD7",
-  "4": "0x42d0f5fE517A0e4aB8be2F3FA7799fB96E30E777"
+  "1":  "0xa42d613D8B8B6cB9507F46A5dd3FD98276570FD7",
```

Comment:
very picky but why does this file have 2 spaces after the colon?

Reply:
it's been resolved.